pkg/cloud/api/doc.go

```diff
@@ -25,6 +25,31 @@ limitations under the License.
 // Resource is used to write version-agnostic code such as Kubernetes-style API
 // translators.
 //
+// Every resource needs to define its all 3 types traits GA, Alpha and Beta
```

Comment:
I think it's easier just to document this in the comment to ImpliedVersion. I've updated my commit comments there on the other PR. We should just replace this text with a reference to the code docstring:

```
See MutableResource.ImpliedVersion() for resource version conversion semantics.
```